src/build/streams.ts

```diff
@@ -17,7 +17,9 @@ import * as stream from 'stream';
 
 import File = require('vinyl');
 
-(Symbol as any).asyncIterator = Symbol.asyncIterator || Symbol('asyncIterator');
+if ((Symbol as any).asyncIterator !== 'undefined') {
```

Comment:
This isn't right, it's string undefined rather than the value undefined, I'll send over a PR